staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/types.go

```diff
@@ -25,5 +25,36 @@ type WebhookAdmission struct {
 	metav1.TypeMeta
 
 	// KubeConfigFile is the path to the kubeconfig file.
+	// +optional
 	KubeConfigFile string
+
+	// ExclusionRules are a list of rules that the webhooks while ignore
+	// +optional
+	ExclusionRules []ExclusionRule //NOTE: using ExclusionRule rather than Rule with name/namespace because don't want to have scope field
+}
+
+type ExclusionRule struct {
```

Comment:
[`NamedRuleWithOperations`](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go#L349) seems pretty close to what is needed here.

It is not clear to me why this omits scope and operation or why it uses kind instead of resource.